doc/user_guide/io.rst

```diff
@@ -578,15 +578,17 @@ Extra loading arguments
 select_type: One of ('spectrum', 'image'). If specified just selected type of
 data is returned. (default None)
 
-index: index of dataset in bcf v2 files, which can hold few datasets (delaut 0)
+index: One of (None, int, "all"). Index of dataset in bcf file, which can hold few datasets.
+Default None value result in first or the only dataset returned (the lowest index available).
+When set to 'all' it result in loading all available datasets.
```

Comment:
Improvement suggestion:
index: one of (None, int, "all"). Allow to select the index of the dataset in the bcf file, which can contains several datasets. Default None value result in loading the first dataset. When set to 'all', all available datasets will be loaded and returned as separate signals.